ros_buildfarm/config/ci_build_file.py

```diff
@@ -94,6 +94,12 @@ def __init__(self, name, data):
             self.underlay_from_ci_jobs = data['underlay_from_ci_jobs']
             assert isinstance(self.underlay_from_ci_jobs, list)
 
+        self.archive_files = []
+        if 'archive_files' in data:
+            self.archive_files = data['archive_files']
+            assert isinstance(self.archive_files, list)
+            assert all(isinstance(path, str) for path in self.archive_files)
```

Comment:
This new configuration option needs to be documents in the rst files.

Reply:
See https://github.com/ros-infrastructure/ros_buildfarm/pull/784/commits/f205f7538c4607ba1825f24f7131ba877d0dab14.